python/nano/test/openvino/tf/test_openvino_quantize.py

```diff
@@ -52,19 +52,24 @@ def test_model_quantize_openvino(self):
         y_hat = openvino_quantized_model(train_examples[:10])
         assert y_hat.shape == (10, 10)
 
-        y_hat = openvino_quantized_model.predict(train_examples, batch_size=5)
-        assert y_hat.shape == (100, 10)
-
         preds = model.predict(train_examples)
         openvino_preds = openvino_quantized_model.predict(train_examples)
         np.testing.assert_allclose(preds, openvino_preds, rtol=1e-2)
 
         # Case 3: with config
-        openvino_quantized_model = InferenceOptimizer.quantize(model,
+        openvino_quantized_model = InferenceOptimizer.quantize(openvino_model,
```

Comment:
why openvino_model

Reply:
to save time, this will only call `trace` once